.github/workflows/agent-language-tests.yml

```
@@ -11,49 +10,29 @@ on:
+    container:
+      image: coasys/ad4m-ci-linux:latest@sha256:3d6e8b6357224d689345eebd5f9da49ee5fd494b3fd976273d0cf5528f6903ab
+    env:
+      npm_config_build_from_source: true
 
-    strategy:
-      matrix:
-        platform: [ubuntu-22.04]
-        node-version: [22.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)